### PR TITLE
Tests for undo redo sort

### PR DESCRIPTION
**Pull request by SpecOps2016**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/UndoCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import seedu.address.model.UserPrefs;
+
+public class UndoCommandTest {
+
+    private final Model model = new ModelManager(getTypicalPatientsHealthContact(), new UserPrefs());
+    private final Model expectedModel = new ModelManager(getTypicalPatientsHealthContact(), new UserPrefs());
+
+
+    @Test
+    public void execute() throws CommandException {
```
Comment (guokweijie) [inline src/test/java/seedu/address/logic/commands/UndoCommandTest.java:20]: can add more tests, eg. test for the edit, list edit case

File: src/test/java/seedu/address/logic/commands/RedoCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+public class RedoCommandTest {
+    private final Model model = new ModelManager(getTypicalPatientsHealthContact(), new UserPrefs());
+    private final Model expectedModel = new ModelManager(getTypicalPatientsHealthContact(), new UserPrefs());
+
+
+    @Test
```
Comment (guokweijie) [inline src/test/java/seedu/address/logic/commands/RedoCommandTest.java:18]: test for the cases we found

**Review by guokweijie — CHANGES_REQUESTED** [review]
read the comments

**Review by Fuzanwenn — APPROVED** [review]
LGTM

**Pull request merged**